LDAR_Sim/testing/unit_testing/test_virtual_world/test_infrastructure/test_update_propagating_params.py

```diff
@@ -48,6 +48,13 @@ def generate_test_prop_params(draw, methods):
         to_ret[cp.METH_SPECIFIC][key] = {}
         for method in methods:
             to_ret[cp.METH_SPECIFIC][key][method] = draw(st.integers(min_value=0, max_value=10000))
+    to_ret[cp.METH_SPECIFIC][
+        Infrastructure_Constants.Sites_File_Constants.SITE_DEPLOYMENT_PLACEHOLDER
+    ] = {}
+    for method in methods:
+        to_ret[cp.METH_SPECIFIC][
+            Infrastructure_Constants.Sites_File_Constants.SITE_DEPLOYMENT_PLACEHOLDER
+        ][method] = {}
```

Comment:
Is this supposed to be a dictionary?

Reply:
Yes, it makes a dictionary with the method names as keys, then the actual values needed.